content/join.hbs

```diff
@@ -25,7 +25,8 @@
 			</ol>
 			<p class="has-text-centered">This list is not exhaustive; other entrance standards may apply.</p>
 
-			<h1>Join in the <span class="semester-text">Spring 2021</span> semester</h1>
+			<h1>If you are a college student, follow the steps below to join in the <span class="semester-text">Spring 2021</span> semester</h1>

```

Comment:
This extra text breaks the styling of the page and is very wordy. First of all, is it really necessary? (I don't see high school students applying for the Spring semester considering they graduate in the fall)